manager/integration/tests/test_backing_image.py

```diff
@@ -694,3 +704,199 @@ def test_volume_wait_for_backing_image_condition(client): # NOQA
     volume_endpoint = get_volume_endpoint(volume2)
     vol2_cksum = get_device_checksum(volume_endpoint)
     assert vol1_cksum == vol2_cksum
+
+
+@pytest.mark.backing_image  # NOQA
+def test_backing_image_min_number_of_replicas(client): # NOQA
+    """
+    Test the backing image minNumberOfCopies
+
+    Given
+    - Create a BackingImage
+    - Update the minNumberOfCopies to 2
+
+    When
+    - The BackingImage is prepared and transferred to manager
+
+    Then
+    - There will be two ready file of the BackingIamge
+
+    Given
+    - Set the setting backing-image-cleanup-wait-interval to 1 minute
+
+    When
+    - Update the minNumberOfCopies to 1
+    - Wait for 1 minute
+
+    Then
+    - There will be one ready BackingImage file left.
+    """
+    bi_url = BACKING_IMAGE_RAW_URL
+    create_backing_image_with_matching_url(
+        client, BACKING_IMAGE_NAME, bi_url)
```

Comment:
Is the default mimNumberOfCopies 3? Can we check the value after the creation as well?

Reply:
the default of `create_backing_image_with_matching_url()` function is 1
this test first create a minNumberOfCopies=1 backingimage
then update it to 2
then check if there are 2 files ready.

```
def create_backing_image_with_matching_url(client, name, url,
                                           minNumberOfCopies=1,
                                           nodeSelector=[], diskSelector=[]):
```

Reply:
I see. Can we check number of copies 1 after `create_backing_image_with_matching_url()`?

Reply:
is already verified
```
    is_ready = False
    for i in range(RETRY_COUNTS):
        bi = client.by_id_backing_image(name)
        if len(bi.diskFileStatusMap) == minNumberOfCopies and bi.currentChecksum != "":
            for disk, status in iter(bi.diskFileStatusMap.items()):
                if status.state == "ready":
                    is_ready = True
                    break
            if is_ready:
                break
        time.sleep(RETRY_INTERVAL)

```